lua/telescope/mappings.lua

```diff
@@ -28,7 +28,9 @@ mappings.default_mappings = config.values.default_mappings or {
       ["<S-Tab>"] = actions.toggle_selection + actions.move_selection_better,
       ["<C-q>"] = actions.send_to_qflist + actions.open_qflist,
       ["<M-q>"] = actions.send_selected_to_qflist + actions.open_qflist,
-      ["<C-l>"] = actions.complete_tag
+      ["<C-l>"] = actions.complete_tag,
+
+      ["<M-w>"] = actions.delete_buffer,
```

Comment:
Why is this mapping global? You should do a `map(mode, key, actions.delete_buffer)` in `attach_mappings`.

Reply:
Do you mean something like this: a38e3ebfe04ffde68bdf9328ff9c229ee667db56? 

> mappings are tricky business. I think it should me implemented as action for future reuse and decrease the complexity of the main function.

That was my original plan but @tami5 suggested to make it global or are you suggesting something else? I am sorry if I am misunderstanding :)

Reply:
Oh, I am sorry, I got it now what you mean. You mean to keep the actions global but mappings local.

Reply:
Global in the sense that it applies to all the pickers. mappings should applied to the picker it self. I guess you can refer git pickers as they have local mappings

Reply:
Got it, thanks for clearing it up!